task_processing/metrics.py

```diff
@@ -0,0 +1,54 @@
+import mock
```

Comment:
I think @jolynch's idea that we move the mesos-specific constants to the executor is good, so we don't need to move metric itself.